jenkins/tox

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# A wrapper around tox to ensure the jenkins environment is setup
+# properly inside a container.
+
+export USER=pbench
+export LOGNAME=pbench
+
+export TERM=dumb
+export PY_COLORS=0
+export NO_COLORS=0
+
+export TOXWORKDIR=/var/tmp/tox
+mkdir -p ${TOXWORKDIR}
+if [[ ${?} -ne 0 ]]; then
+    exit 1
+fi
+export HOME=${TOXWORKDIR}
+export XDG_CONFIG_HOME=${TOXWORKDIR}/.config
+
+export PATH=${PATH}:/usr/sbin  # For /usr/sbin/ip, used by pbench-register-tool
```

Comment:
Seems like we really need to do this in `agent/profile` as well, by just adding a `_pbench_pathins /usr/sbin` before line 36.

And this would be better served in the `agent/util-scripts/unittests` since it is not necessarily a "jenkins" thing.

Reply:
I'm going to defer this; tracked in #2383.